tgui/packages/tgui/interfaces/MafiaPanel.js

```diff
@@ -131,7 +142,8 @@ export const MafiaPanel = (props, context) => {
             <Section
               title="Notes">
               <Table>
-                {!!roleinfo.action_log && roleinfo.action_log.map(log_line => (
+                {roleinfo !== undefined && !!roleinfo.action_log
+                && roleinfo.action_log.map(log_line => (
```

Comment:
All of this could be written as just `roleinfo?.action_log?.map(log_line => ...`

Reply:
👍